masterlist.yaml

```
@@ -10465,6 +10867,7 @@ plugins:
       - <<: *patchProvided
         subs: [ 'Adamant: A Perk Overhaul' ]
         condition: 'active("Adamant.esp") and not active("Triumvirate - Adamant Compatibility Patch.esp")'
+      - *patch3rdParty_SRPC
```

Make sure the aliases are in alphabetical order.

```suggestion
      - *patch3rdParty_SRPC
      - <<: *patchProvided
        subs: [ 'Adamant: A Perk Overhaul' ]
        condition: 'active("Adamant.esp") and not active("Triumvirate - Adamant Compatibility Patch.esp")'
```